be/src/service/internal_service.cpp

```diff
@@ -707,6 +707,7 @@ void PInternalServiceImpl::_get_column_ids_by_tablet_ids(
         int64_t index_id = param.indexid();
         auto tablet_ids = param.tablet_ids();
         std::set<std::set<int32_t>> filter_set;
```

Comment:
How about simplify it by modifying this line to `std::set<std::vector<TabletColumn>> filter_set;`.